src/Pact/Eval.hs

```diff
@@ -491,29 +489,27 @@ loadInterface i body info gas0 = do
       mangleDefs (_interfaceName i) <$> idefs
   let md = ModuleData (MDInterface i) evaluatedDefs mempty
   installModule True md Nothing
-  return (gas1,md)
+  return md
 
 -- | Retrieve map of definition names to their corresponding terms
 -- and compute their gas value
 --
 collectNames
-  :: Gas
-    -- ^ initial gas value
```

Comment:
So nice that you are able to get rid of these initial gas values! Were they there in the code base as a vestigial thing that existed before `Gas` got added to the `EvalState`, and we never fully migrated off of them until now?